script/add-codeql-to-path

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(dirname "$0")/.."
+
+gh extensions install github/gh-codeql
+gh codeql download
+
+export PATH=$PWD:$PATH
+
+chmod +x codeql
```

Comment:
I'd suggest putting `codeql` in the `script` directory to keep things a bit cleaner. Git will remember the `chmod +x` for us so you don't need this line.